spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -539,6 +539,14 @@ def create_shortcuts(self):
                                        name='Blockcomment', parent=self)
         unblockcomment = config_shortcut(self.unblockcomment, context='Editor',
                                          name='Unblockcomment', parent=self)
+        transform_uppercase = config_shortcut(self.transform_to_uppercase,
+        							            context='Editor',

```

Comment:
I don't know what you do with your last commit, but this is misaligned again. Please fix.